db/migrate/20161003091412_add_exclusive_lock_indexes.rb

```diff
@@ -0,0 +1,7 @@
+class AddExclusiveLockIndexes < ActiveRecord::Migration
+  def change
+    add_index(:foreman_tasks_locks, [:name, :resource_type, :resource_id, :exclusive],
+              :name => 'index_foreman_tasks_locks_all_lock_properties')
+    add_index(:foreman_tasks_locks, :task_id)
```

Comment:
I've revisited this patch to just eliminate the missed indexes, when the migration happened on rails < 4.2